fenix/app/src/main/java/org/mozilla/fenix/onboarding/view/JunoOnboardingPageType.kt

```diff
@@ -0,0 +1,14 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.onboarding.view
+
+/**
+ * Model for different types of Onboarding Pages.
+ */
+enum class JunoOnboardingPageType {
```

Comment:
This is just a thought. I think we should have a plan on whether or not we should drop the 'Juno' prefix here. It doesn't have to happen right now, but if this goes beyond just experimentation and becomes the default first run onboarding, then we should ideally drop the these types of project prefixes because they won't provide much context a year or two from when the project phase ended. For example, we have had MR1/MR2 2021 and 2022 but once the feature releases for awhile, the meaning of those major release and the contents within them simply becomes a historical event with no context for anyone to really grasp what went into each release.

Reply:
This is a good point @gabrielluong. I've come across a few legacy project names in the code that are quite confusing without the context, definitely something we should be mindful of. 
@rahulsainani agree that dropping it once it becomes the default is a good way forward - for now I think the prefix is useful for identifying the 'new' onboarding elements